changelog/fragments/mv-EnqueueRequestForAnnotation.yaml

```diff
@@ -0,0 +1,8 @@
+entries:
+    - description: >
+        Remove the implementation for `EnqueueRequestForAnnotation` handler from sdk repository, and
+        reference it from operator-lib instead.
+      kind: "change"
+  
+      # Is this a breaking change?
+      breaking: false
```

Comment:
@hasbro17 Do you think this is a breaking change? Im not entirely sure about it, since there is no much difference in the handler implementation itself.

Reply:
Pretty sure this is not breaking. I checked that the previous implementation used the same formatting for the annotation key/value pairs.

Reply:
Actually, this is a breaking removal.

The migration would be to import this from `operator-lib`